### ISSUE TITLE
Support for colors

### ISSUE THREAD

**Issue by woutersteven2020** [mnahkies/ng-qrcode#40]
Good morning, 

Great job on the QR code, i like that it supports the center image.

Other packages support specifying the light and dark color of the QR code. How's that possible with this package?

For example:

<qr-code [value]="address" size="120" errorCorrectionLevel="H" margin="1" centerImageSrc="assets/images/logoicon-qr.png" [darkcolor]="'**#2a3042**'"></qr-code>

It's often used for branding, so that in this case, the black squares become dark blue.

**Comment by mnahkies** [comment]
Good idea - I've included this in https://github.com/mnahkies/ng-qrcode/releases/tag/v5.2.0
